samples/csharp/common/ConsoleHelper.cs

```diff
@@ -45,6 +45,30 @@ public static void PrintRegressionMetrics(string name, RegressionEvaluator.Resul
             Console.WriteLine($"*************************************************");
         }
 
+        public static void PrintMulticlassClassificationFoldsAverageMetrics(
+                                         string algorithmName,
+                                         (MultiClassClassifierEvaluator.Result metrics,
+                                          ITransformer model,
+                                          IDataView scoredTestData)[] crossValResults
+                                                                           )
+        {
+            var metricsInMultipleFolds = crossValResults.Select(r => r.metrics);
+
+            var microAccuracies  = metricsInMultipleFolds.Select(m => m.AccuracyMicro);
+            var macroAccuracies  = metricsInMultipleFolds.Select(m => m.AccuracyMacro);
+            var logLoss          = metricsInMultipleFolds.Select(m => m.LogLoss);
+            var logLossReduction = metricsInMultipleFolds.Select(m => m.LogLossReduction);
+
+            Console.WriteLine($"**************************************************************************");
+            Console.WriteLine($"*       Metrics for {algorithmName} Multi-class Classification model      ");
+            Console.WriteLine($"*-------------------------------------------------------------------------");
+            Console.WriteLine($"*       Average MicroAccuracy:    {microAccuracies.Average():0.##}");
+            Console.WriteLine($"*       Average MacroAccuracy:    {macroAccuracies.Average():0.##}");
+            Console.WriteLine($"*       Average LogLoss:          {logLoss.Average():#.##}");
+            Console.WriteLine($"*       Average LogLossReduction: {logLossReduction.Average():#.##}");
```

Comment:
The command-line (maml) prints in this format:
```
Accuracy(micro-avg): 0.690777 (0.0106)
Accuracy(macro-avg): 0.447108 (0.0267)
Log-loss:           1.852149 (0.1039)
Log-loss reduction: 9.075382 (5.1809)

---------------------------------------
Physical memory usage(MB): 2825
Virtual memory usage(MB): 3564
6/4/2018 9:32:19 PM        Time elapsed(s): 138
```

The first value is the average, and second value in the parenthesis is the standard deviation of the CV folds. CV is a costlier mode vs TrainTest (n-times slower). The main gain in CV mode (vs TrainTest) is its better estimate of your metric and the error in the metric.

Can we print the standard deviation? 

---

Another interesting choice is a 95% [confidence interval](https://en.wikipedia.org/wiki/Confidence_interval#Basic_steps):
```
Accuracy(micro-avg): 0.690777 ± 0.0464
```

Basic formula: `Avg` ± `1.96 * StdDev / SQRT(numCVFolds)`. There are simplifying assumptions made, for instance assuming a normal distribution, but the utility is still there.

In the end, I use maml's printed standard deviation to calculate error bars, which lets me have a general idea if two runs are the same or one is statistically significantly better.

Reply:
I added the standard deviation and 95% confidence interval for each metric.

This is how it'd be printed:
![image](https://user-images.githubusercontent.com/1712635/47969858-673f7a00-e032-11e8-87a8-ad32a58b30a7.png)

Canyou check if the following formulas in code right?

```
        public static double CalculateStandardDeviation (IEnumerable<double> values)
        {
            double average = values.Average();
            double sumOfSquaresOfDifferences = values.Select(val => (val - average) * (val - average)).Sum();
            double standardDeviation = Math.Sqrt(sumOfSquaresOfDifferences / values.Count());
            return standardDeviation;
        }

        public static double CalculateConfidenceInterval95(IEnumerable<double> values)
        {
            double confidenceInterval95 = 1.96 * CalculateStandardDeviation(values) / Math.Sqrt(values.Count());
            return confidenceInterval95;
        }

```

Reply:
Do we have either a standard deviation or preferably a standard-error (StdErr) function in our TLC stats package?  (or the [built-in](https://docs.microsoft.com/en-us/dotnet/api/system.data.objects.entityfunctions.standarddeviation?view=netframework-4.7.2))

If it's simple to call, it would be nice to re-use code. Someone likely has already thought of the corner cases. 

The confidence interval would be the full range of: `0.690777 ± 0.0464`. Also a leading `0` w/ four digits would look nice: `0.0150` (current is `.015`).